rosdep/python.yaml

```
@@ -6304,6 +6304,17 @@ python3-datadog-pip:
   ubuntu:
     pip:
       packages: [datadog]
+python3-dateutil:
+  arch: [python-dateutil]
+  debian: [python3-dateutil]
+  fedora: [python3-dateutil]
+  gentoo: [dev-python/python-dateutil]
+  nixos: [python310Packages.python-dateutil]
+  opensuse: [python3-python-dateutil]
+  osx:
+    pip:
+      packages: [python-dateutil]
+  ubuntu: [python3-dateutil]
```

I'm not sure why the suggestions test missed this, but the package is available for RHEL 7 and 8:
* `7`: https://packages.fedoraproject.org/pkgs/python3-dateutil/python36-dateutil/
* `8`: https://repo.almalinux.org/almalinux/8/BaseOS/x86_64/os/Packages/python3-dateutil-2.6.1-6.el8.noarch.rpm

```suggestion
  rhel: ['python%{python3_pkgversion}-dateutil']
  ubuntu: [python3-dateutil]
```

---

Ah, I see. The suggestion code worked, but RHEL 7 uses a different name, which is why I'm suggesting the use of `%{python3_pkgversion}` here.